### PR TITLE
refactor: rename 'ToolchainEvent' type to 'SocialEvent'

### PR DESCRIPTION
**Pull request by xcoulon**
see https://github.com/codeready-toolchain/api/pull/298

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -27,4 +27,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220405142115-a332d4046e57
```
Comment (alexeykazakov) [inline go.mod:30]: Reminder.

Comment (xcoulon) [reply go.mod]: addressed in [d8b5e30](https://github.com/codeready-toolchain/host-operator/pull/622/commits/d8b5e301ec13ef8f865584a09fd65cf89e208126)

**Review by MatousJobanek — COMMENTED** [review]
@xcoulon please update also the `PROJECT` file as well as `config/crd/kustomization.yaml`

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**